### DISCUSSION TITLE
FFmpeg 24 bit example ?

### DISCUSSION THREAD

**Discussion by SantiagoSotoC** [snapcast/snapcast#1255] [Q&A]
I would like an example of how to send 24 bit audio from ffmpeg to snapserver, my attempts were not successful, it is to add 24 bit support to music assistant

**Comment by rvk** [comment]
Considering quite a few people use Music Assistant nowadays and SantiagoSotoC is maintainer of its Snapcast integration, it would be very nice if this got answered. I found this question while trying to find out how to get Music Assistant to use 24bit with Snapcast, which seems not to be possible currently.

**Reply by SantiagoSotoC** [reply]
I saw that with gstreremer it seems to work, they use a format that is not available in ffmpeg. It's a big dependency to add it to ma. 
Now I don't remember exactly the format and but I only get static when using 24/32 bits.

**Comment by rwjack** [comment]
https://github.com/snapcast/snapcast/pull/1532
